src/ui/components/ProtocolTimeline.tsx

```diff
@@ -70,13 +79,17 @@ const Spans = ({
 export default function ProtocolTimeline() {
   const loadedRegions = useAppSelector(getLoadedRegions);
   const { value: showProtocolTimeline } = useFeature("protocolTimeline");
-  const firstPaint = gPaintPoints[0];
-  const lastPaint = gPaintPoints[gPaintPoints.length - 1];
+  const firstPaint = imperitavelyGetPaintPointForTime(0)!;
```

Comment:
I guess we could do like a `getNextPaintAfter` and pass in the last paint we have seen?

Reply:
Nah, this is fine as is.